plugins/ruby-foreman-remote-execution/debian/control

```diff
@@ -3,7 +3,8 @@ Section: ruby
 Priority: extra
 Maintainer: Stephen Benjamin <stephen@redhat.com>
 Build-Depends: debhelper (>= 8.0.0), git | git-core, foreman-assets (>= 1.20.0~rc1), foreman (>= 1.25.0~rc1),
-  foreman-nulldb (>= 1.25.0~rc1), ruby-foreman-tasks (>= 0.15.1), ruby-foreman-deface
+  foreman-nulldb (>= 1.25.0~rc1), ruby-foreman-tasks (>= 0.15.1), ruby-foreman-deface,
+  ruby-dynflow (>= 1.0.2), ruby-dynflow (<< 2.0.0)
```

Comment:
For core plugins, dynflow is provided by the foreman (core) package, so this line needs to get removed.

Reply:
So if we need a minimum version (1.0.2), we have to hope foreman (core) package provides it?

Reply:
Yes, but if the core package in the build process doesn't bring it, the build would fail. So we detect this at least with the latest core package that's available for the target branch.

Reply:
BTW, 1.0.2 was release two years ago, so I'd say this is super-safe. ;)